crates/driver/src/tests/setup/mod.rs

```diff
@@ -73,33 +72,71 @@ pub enum Score {
     RiskAdjusted { success_probability: f64 },
 }
 
-#[serde_as]
-#[derive(Debug, Clone, PartialEq, serde::Serialize)]
-#[serde(rename_all = "camelCase", tag = "kind")]
-pub enum FeePolicy {
-    #[serde(rename_all = "camelCase")]
-    Surplus { factor: f64, max_volume_factor: f64 },
-    #[serde(rename_all = "camelCase")]
-    Volume { factor: f64 },
-}
-
-impl FeePolicy {
-    pub fn to_json_value(&self) -> serde_json::Value {
-        match self {
-            FeePolicy::Surplus {
-                factor,
-                max_volume_factor,
-            } => json!({
-                "surplus": {
-                    "factor": factor,
-                    "maxVolumeFactor": max_volume_factor
-                }
-            }),
-            FeePolicy::Volume { factor } => json!({
-                "volume": {
-                    "factor": factor
-                }
-            }),
+pub mod fee {
```

Comment:
I think this one needs a bit of clean up. Either revert or move this module into its own file please.

Reply:
Done